### PR TITLE
Update versioning sample

### PR DESCRIPTION
**Pull request by lailabougria**
Closes https://github.com/Particular/EnhancementReleases/issues/17

Updates the versioning sample to match the documentation as described in https://github.com/Particular/docs.particular.net/pull/5435

### CODE REVIEW

File: samples/versioning/Core_7/V2.Receiver/V2.Receiver.csproj [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    </PropertyGroup>
+
+    <ItemGroup>
+      <PackageReference Include="NServiceBus" Version="7.5.0" />
+      <PackageReference Include="NServiceBus.Transport.Msmq" Version="1.1.1" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <ProjectReference Include="..\V2.Contracts\Versioning.Contracts.csproj" />
+      <ProjectReference Include="..\V2.Messages\V2.Messages.csproj" />
```
Comment (WojcikMike) [inline samples/versioning/Core_7/V2.Receiver/V2.Receiver.csproj:16]: This project does not exist, I think it should be removed right?

File: samples/versioning/Core_7/V2.Subscriber/V2.Subscriber.csproj [hunk 2]
```diff
@@ -5,7 +5,8 @@
     <LangVersion>7.3</LangVersion>
   </PropertyGroup>
   <ItemGroup>
-    <ProjectReference Include="..\V1.Messages\V1.Messages.csproj" />
+    <ProjectReference Include="..\V2.Contracts\Versioning.Contracts.csproj" />
+    <ProjectReference Include="..\Versioning.Messages\Versioning.Messages.csproj" />
```
Comment (WojcikMike) [inline samples/versioning/Core_7/V2.Subscriber/V2.Subscriber.csproj:9]: Versioning.Messages and V2.Messages should be removed from here right?

**Review by WojcikMike — CHANGES_REQUESTED** [review]

File: samples/versioning/Core_7/V2.Subscriber/V2.Subscriber.csproj (hunk 2 above)
Comment (lailabougria) [reply samples/versioning/Core_7/V2.Subscriber/V2.Subscriber.csproj:9]: Yep, thanks, must have been some leftovers left when i removed the project :/

Comment (lailabougria) [reply samples/versioning/Core_7/V2.Subscriber/V2.Subscriber.csproj:9]:
```suggestion
```

**Review by dvdstelt — APPROVED** [review]

**Pull request merged**